de.peeeq.wurstscript/src/main/java/de/peeeq/wurstscript/translation/imoptimizer/InitFunctionCleaner.java

```diff
@@ -0,0 +1,59 @@
+package de.peeeq.wurstscript.translation.imoptimizer;
+
+import de.peeeq.wurstscript.jassIm.*;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class InitFunctionCleaner {
+
+    public static void clean(ImProg prog) {
+        List<ImFunction> initFuncs = prog.getFunctions().stream().filter(func -> func.getName().startsWith
+                ("init_") && func.getBody().size() == 1).collect(Collectors.toList());
+
+        initFuncs.forEach(func -> {
+            ImVar globalBridge = null;
+            for (ImVar global : prog.getGlobals()) {
+                if (global.getName().equals("ref_function_" + func.getName())) {
```

Comment:
I don't like using the name here. Names should not matter in the intermediate language.

Reply:
What to use then?